Sources/ArcGISToolkit/Common/AttachmentList.swift

```diff
@@ -50,7 +50,7 @@ struct AttachmentRow: View  {
                 .frame(maxWidth: .infinity, alignment: .leading)
             }
             .onTapGesture {
-                if attachmentModel.attachment.loadStatus == .loaded {
+                if attachmentModel.loadStatus == .loaded {
```

Comment:
This isn't directly related to the bug; this should've been accessing the model's property.